src/ol/extent.js

```diff
@@ -826,7 +826,7 @@ export function applyTransform(extent, transformFn, opt_extent, opt_stops) {
 export function wrapX(extent, projection) {
   const projectionExtent = projection.getExtent();
   const center = getCenter(extent);
-  if (projection.canWrapX() && (center[0] < projectionExtent[0] || center[0] > projectionExtent[2])) {
+  if (projection.canWrapX() && (center[0] < projectionExtent[0] || center[0] >= projectionExtent[2])) {
```

Comment:
Please elaborate why you think this is necessary.

Reply:
If you have two extents with centers at -180 and +180 they should be moved to the same real world (as +540 would be) but the with > that didn't happen, the graticule thought it had different extents and went into a loop.

Reply:
Ok, thanks for the explanation. I think this fine for extent centers, but not for coordinates.